CamMerrill/Shackelford
# Improved line measuring

## Summary
Creates a new tool called the BP_LineMeasure that acts a little bit like the line level in real life, except it has the added bonus feature of giving a readout on length (height) of the distance between the ball and the point at which the red spline collides with the ground.

## Notes
For some reason the HeightLine spline static mesh can potentially not be visible when we move the ball around. It will reappear if you wiggle the ball, but this is still an annoying and persistent issue.

## Testing
1) Add the BP_LineMeasure to the world
2) Add a BP_Stake to the world
3) Pickup the BP_LineMeasure, attach to the stake
4) Move around, and assert with your eyes that once the line is level, the word level appears and the line highlights blue
5) Move around the ball, assert that the distances are logical distances for the height readout
